schemas.js

```diff
@@ -3,6 +3,7 @@
 const Joi = require('joi');
 
 const base = {
+  'liveMode': Joi.string().empty('').default('y').valid(['y', 'n']),
```

Comment:
Thinking about this in line with my above comment... how about having `liveMode` default as `n`. Any reason for preference for `y`?